src/include/wiredtiger.in

```diff
@@ -270,7 +270,8 @@ struct __wt_cursor {
 	 */
 	/*!
 	 * Return the ordering relationship between two cursors: both cursors
-	 * must have the same data source and have valid keys.
+	 * must have the same data source and have valid keys. (When testing
+	 * only for equality, WT_CURSOR::compare_equality may be faster.)
```

Comment:
Thanks, fixed.

There's no other documentation here in WT_CURSOR::compare (there's separate WT_CURSOR::compare_equal documentation, of course).

WT_CURSOR::compare_equal returns 0/!0 in the parameter, the function returns error status.